app/views/sections/view-solution-description.njk

```diff
@@ -0,0 +1,32 @@
+{% from '../components/view-question-title.njk' import viewQuestionTitle %}
+{% from '../components/view-question-data-text.njk' import viewQuestionDataText %}
+{% from '../components/view-question-data-link.njk' import viewQuestionDataLink %}
+
+{% macro viewSolutionDescription(section) %}
+  {% if section %}
+    <div data-test-id="view-solution-description">
+      <div class="bc-c-title-block">
+        <h3 class="nhsuk-u-margin-bottom-2">Solution description</h3>
+      </div>
+      <div class="nhsuk-u-margin-bottom-4 nhsuk-u-padding-left-3">
+        {% if section.answers.summary %}
+          <div data-test-id="view-section-question-summary">
+              {{ viewQuestionTitle('summary', 'Summary') }}
+              {{ viewQuestionDataText('summary', section.answers.summary) }}
+          </div>
+        {% elseif section.answers.description %}
```

Comment:
I don't think this should be an elseif. It is possible to show both. It should depend on what data is provided from the API. If it is there then show it

Reply:
ok it wasn't in the design and seemed a bit much to have the summary and description in there but can add back

Reply:
it should have both if both have been provided by the supplier. I think we are just a little ahead of the design at this point

Reply:
whoop whoop 💃